.gitattributes

```diff
@@ -1,2 +1,2 @@
-dist/** -diff linguist-generated=true
+**/dist/** -diff linguist-generated=true
```

Comment:
This _should_ be the same.

- `/dist/**` would be dir paths inside the root dir dist only
- `dist/**` would be dir paths inside **any** dist
- `**/dist/**` would be "dir paths inside **any** dist that is inside a dir (which is included in the one above).

Reply:
Before the index.js diff was displayed and not detected as generated 🤷

Reply:
Yeah, probably linguist that doesn't do this correctly :)